packages/web-components/src/horizontal-scroll/index.ts

```diff
@@ -2,6 +2,15 @@ import { customElement } from '@microsoft/fast-element';
 import { HorizontalScroll, HorizontalScrollTemplate as template } from '@microsoft/fast-foundation';
 import { ActionsStyles, HorizontalScrollStyles as styles } from './horizontal-scroll.styles';
 
+/**
+ * The Fluent Horizontal Scroll. Implements {@link @microsoft/fast-foundation#HorizontalScroll},
+ * {@link @microsoft/fast-foundation#HorizontalScrollTemplate}
+ *
+ *
+ * @public
+ * @remarks
+ * HTML Element: \<fluent-flipper\>
```

Comment:
Can you update the element name on line 12?

Reply:
This had already been fixed. I had an out of date repo.